utils/src/ooinstall/cli_installer.py

```diff
@@ -510,7 +510,7 @@ def error_if_missing_info(oo_cfg):
         sys.exit(1)
 
     # Lookup a variant based on the key we were given:
-    if not oo_cfg.settings['variant']:
+    if not 'variant' in oo_cfg.settings:
```

Comment:
maybe use `if oo_cfg.settings.get('variant') is not None' here, so that you will also catch the case where the variant key is present in the config, but does not have a value set.